scripts/get_build_parameters.mjs

```diff
@@ -49,7 +49,7 @@ export function getBuildParameters(buildParameters) {
   platform ??= 'browser';
   buildMode ??= 'debug';
 
-  return {platform, buildMode};
+  return {platform, buildMode, cordovaPlatform: platform === 'macos' ? 'osx' : platform};
```

Comment:
This is now exposing Cordova stuff to code that doesn't care about Cordova. Let's revert.
My suggestion was to have a separate function that maps our platform to a cordova platform.
This way the code that cares about cordova can call something like `toCordovaPlatform(platform)`